ddl/table.go

```diff
@@ -449,12 +450,16 @@ func onRebaseAutoID(store kv.Storage, t *meta.Meta, job *model.Job) (ver int64,
 		job.State = model.JobStateCancelled
 		return ver, errors.Trace(err)
 	}
-	tblInfo.AutoIncID = newBase
 	tbl, err := getTable(store, schemaID, tblInfo)
 	if err != nil {
 		job.State = model.JobStateCancelled
 		return ver, errors.Trace(err)
 	}
+	newBase, err = getNextAutoID(w.sessPool, tbl, newBase)
```

Comment:
I think just check:
```
if tblInfo.AutoIncID < newBase {
	tblInfo.AutoIncID = newBase
}
```

is ok. tblInfo.AutoIncID is only used by getAutoIncrementID

Reply:
em... In this PR: https://github.com/pingcap/tidb/pull/10207/files, You will think AutoIncID is not useful after create table

Reply:
After condideration, there is no logical code, only add comment.